packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -1,24 +1,35 @@
+@import "../../assets/styles/header";
+
 /**
- * CSS Custom Properties
- *
- * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-tip-manager-height: The maximum height of the component.
- * @prop --calcite-tip-max-width: The maximum width of a slotted `calcite-tip` within the component.
- */
+* CSS Custom Properties
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-max-width: [Deprecated] No longer necessary. 
```

Comment:
Shouldn't the deprecated ones point to the matching props to use? Or why are these no longer necessary?

Reply:
These are both documented. My guess is they were requested by end users. So this is probably an edge case where we should keep them. Maybe rename to `--calcite-tip-max-size-x` and `--calcite-tip-max-size-y`

Reply:
In this particular case, `--calcite-tip-max-width` is used to set the max-width directly on slotted `tip`s. Wouldn't we want to deprecate it in favor of rules targeting `tip`s to set the max-width? It'd be one less prop to maintain.

Reply:
Yeah, in these cases a user could also just set these css rules directly on the Tip element correct, as it is slotted into the Tip Manager? 

Anecdotally, Tip / Tip Manager is one of our oldest and most seldom used components, and is slated for deprecation once Carousel lands. I think it's relatively safe to deprecate these - they'll continue to work and we don't have to add or support new css properties.